src/components/utils/ControlledAlert/useAlertStyles.tsx

```diff
@@ -37,30 +39,81 @@ const getAlertVariantIcon = (alertVariant: AlertVariant) => {
   }
 };
 
+/** Solid Alert Options **/
+
+const getSolidAlertVariantColor = (alertVariant: AlertVariant) => {
+  switch (alertVariant) {
+    case 'success':
+      return 'green-500' as const;
+    case 'info':
+      return 'blue-400' as const;
+    case 'warning':
+      return 'yellow-600' as const;
+    case 'error':
+      return 'red-400' as const;
+    case 'default':
+    default:
+      return 'blue-400' as const;
+  }
+};
+
+const getSolidAlertVariantBackgroundColors = (alertVariant: AlertVariant) => {
+  switch (alertVariant) {
+    case 'success':
+      return 'green-500-10' as const;
+    case 'info':
+      return 'blue-400-10' as const;
+    case 'warning':
+      return 'yellow-500-10' as const;
+    case 'error':
+      return 'red-500-10' as const;
+    case 'default':
+    default:
+      return 'gray-100' as const;
+  }
+};
+
+const getSolidAlertVariantIcon = (alertVariant: AlertVariant) => {
+  switch (alertVariant) {
+    case 'success':
+      return 'check-circle' as const;
+    case 'warning':
+    case 'error':
+      return 'alert-circle-filled' as const;
+    case 'info':
+      return 'info' as const;
+    case 'default':
+    default:
+      return null;
+  }
+};
```

Comment:
`[suggestion]` Having 3 functions for the alert variant styles feels a little strange to me. I wonder if we can bundle all of the styles into a single function since the variants themselves are the same?

```ts
// I think there's keys in the theme that we could use, I don't remember the exact names though
type SolidVariantTheming = { 
  color: Theme['colors'], 
  bgColor: Theme['colors], 
  icon: Theme['icons]
}

// altername names: getSolidAlertVariantStyles, getSolidAlertVariantConfig, etc.
const getSolidAlertVariantTheming = (variant: AlertVariant) => {
  switch (variant) {
    case 'success': 
      return { color: 'green-500', bgColor: 'green-500-10', icon: 'check-circle' }
    // ...other cases
  }
}
```

Thoughts?